src/azul/time.py

```diff
@@ -106,3 +109,42 @@ def parse_http_date(http_date: str, base_time: Optional[float] = None) -> float:
         return http_date.timestamp()
     else:
         return base_time + float(http_date)
+
+
+dcp2_datetime_format = '%Y-%m-%dT%H:%M:%S.%f%z'
+
+
+def to_dcp2_datetime_string(d: datetime) -> str:
+    """
+    Convert a tz-aware (UTC) datetime into a '2020-01-01T00:00:00.000000Z'
+    formatted string.
+
+    >>> from datetime import timezone
+    >>> to_dcp2_datetime_string(datetime(2020, 1, 1, tzinfo=timezone.utc))
+    '2020-01-01T00:00:00.000000Z'
+
+    >>> to_dcp2_datetime_string(datetime(2020, 1, 1))
+    Traceback (most recent call last):
+    ...
+    azul.RequirementError: 2020-01-01 00:00:00
+    """
+    require(str(d.tzinfo) == 'UTC', d)
+    date_string = datetime.strftime(d, dcp2_datetime_format)
+    assert date_string.endswith('+0000'), date_string
+    return date_string[:-5] + 'Z'
+
+
+def from_dcp2_datetime_string(s: str) -> datetime:
```

Comment:
```suggestion
def parse_dcp2_datetime(s: str) -> datetime:
```

The `to_` and `from_` convention really only works for methods, when there is something to left of them. As in Foo.from_bar